<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewRoomDialog</class>
 <widget class="QDialog" name="addNewRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="roomNumberLabel">
         <property name="text">
          <string>Room number</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="roomNumberLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="roomNameLabel">
         <property name="text">
          <string>Room name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="roomNameLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="priceLabel">
         <property name="text">
          <string>Price</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="priceLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="numberOfBedsLabel">
         <property name="text">
          <string>Number of beds</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="numberOfBedsLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="roomTypeLabel">
         <property name="text">
          <string>Room type</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="roomTypeLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="additionalNotesLabel">
         <property name="text">
          <string>Additional notes</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="balconyLabel">
         <property name="text">
          <string>Balcony</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="balconyCheckBox"/>
       </item>
       <item row="5" column="1">
        <widget class="QTextEdit" name="additionalNotes_textEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Add room</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
